test/index.spec.js

```diff
@@ -84,4 +84,23 @@ describe('immutableStateInvariantMiddleware', () => {
       dispatch({type: 'SOME_ACTION', x});
     }).toNotThrow();
   });
+
+  it('respects "ignore" option', () => {
+    const middlewareIgnore = (next) => {
```

Comment:
oh nice! yea i like that better.  allows the tests to read more easily.  will update.